test/cartesian/getEquidistantTicks.spec.ts

```diff
@@ -0,0 +1,40 @@
+// unit test for getEquidisantTicks
+
+import _ from 'lodash';
```

Comment:
nit: can we remove the dependency from the test by using a for loop below? 

or else at least import the individual lodash dependency as we're trying to do elsewhere

Reply:
Good catch, thank you.